### PR TITLE
Latest files from Crowdin

### PR DESCRIPTION
**Pull request by zapbot**
Autogenerated PR from zapbot

### CODE REVIEW

File: src/lang/Messages_zh_CN.properties [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # labels that are not really expected to be translated, such as the ports at the end of this file
 
 acsrf.api.action.addOptionToken=\u6dfb\u52a0\u4e00\u4e2a\u53cdCSRF\u4ee4\u724c\u7ed9\u5b9a\u540d\u79f0\uff0c\u9ed8\u8ba4\u542f\u7528
-acsrf.api.action.removeOptionToken=Removes the anti-CSRF token with the given name
+acsrf.api.action.removeOptionToken=Elimina el token CSRF anti con el nombre facilitado
```
Comment (thc202) [inline src/lang/Messages_zh_CN.properties]: This is Spanish. I wonder if there's something wrong with Crowdin export?? Following files have also mixed translations.

Comment (psiinon) [inline src/lang/Messages_zh_CN.properties]: We should stop merging Crowdin translations until we're sure they're ok :/

Comment (thc202) [inline src/lang/Messages_zh_CN.properties]: Export is fine, it's the translations that are wrong, a lot of translations were added to the wrong languages, https://crowdin.net/project/owasp-zap/activity_stream

Checking if we can revert that easily...

Comment (thc202) [inline src/lang/Messages_zh_CN.properties]: It seems there's an Undo button.

Comment (thc202) [inline src/lang/Messages_zh_CN.properties]: Translations reverted.

Comment (kingthorin) [inline src/lang/Messages_zh_CN.properties]: Thanks @thc202

**Pull request merged**